### PR TITLE
Trigger visual test regression after create pr to development branch   release/v2.0

### PR DESCRIPTION
**Pull request by pawulonhehe**

### CODE REVIEW

File: billennium/package.json [hunk 1]
```diff
@@ -19,14 +19,17 @@
     "@reduxjs/toolkit": "^1.8.3",
     "axios": "^0.27.2",
     "core-js": "^3.6.5",
+    "cucumber": "^6.0.7",
     "date-fns": "^2.28.0",
     "joi": "^17.6.0",
     "react": "18.1.0",
+    "react-dev-utils": "^12.0.1",
```
Comment (polubis) [inline billennium/package.json:26]: This should be dev dependency I think

File: billennium/package.json [hunk 2]
```diff
@@ -19,14 +19,17 @@
     "@reduxjs/toolkit": "^1.8.3",
     "axios": "^0.27.2",
     "core-js": "^3.6.5",
+    "cucumber": "^6.0.7",
```
Comment (polubis) [inline billennium/package.json:22]: This too

**Review by polubis — CHANGES_REQUESTED** [review]
Please answer comments

**Comment by pawulonhehe** [comment]
working, even without node-modules

**Pull request closed without merging**